services/cnv-qe/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- sarahbx
```

Comment:
Are you the only member of this team?

Reply:
I am not, but others are not part of openshift org yet. I have provided them instructions, and plan to add them afterwards.